content/blog/2022/12/2022-12-05-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,166 @@
+* Jenkins.io is now utilizing Algolia v3 for its search feature.
+* Jenkins in Google Summer of Code 2023, preparations to help new contributors prepare effective proposals.
+
+== Governance Update
+
+Contributed by: https://github.com/MarkEWaite[Mark Waite]
+
+The 2022 Jenkins elections are complete.
+Thanks to Kevin Martens for the https://www.jenkins.io/blog/2022/11/17/jenkins-election-candidates-and-results/[blog post] announcing the results of the election and thanks to Damien Duportal for running the election process.
+We are glad to have received 65 registrations to vote in the election.
+That is less than the 81 registered voters we had in 2021.
+
+Elected officers and members of the Jenkins governance board will begin their service December 3, 2022.
+Special thanks to the members of the governance board that have completed their two-year service:
+
+* https://www.jenkins.io/blog/authors/halkeye/[Gavin Mogan]
+* https://www.jenkins.io/blog/2021/03/22/governance-updates/#about-ewelina-wilkosz[Ewelina Wilkosz]
+
+Congratulations to the new members of the governance board.
+
+* https://www.jenkins.io/blog/authors/uhafner/[Ullrich Hafner]
+* https://www.jenkins.io/blog/authors/notmyfault/[Alexander Brandes]
+
+Congratulations also to the newly elected officers:
+
+* Release Officer - https://www.jenkins.io/blog/authors/timja/[Tim Jacomb]
+* Events Officer - https://www.jenkins.io/blog/authors/alyssat/[Alyssa Tong]
+* Security Officer - https://www.jenkins.io/blog/authors/wadeck/[Wadeck Follonier]
+* Documentation Officer - https://www.jenkins.io/blog/authors/kmartens27/[Kevin Martens]
+* Infrastructure Officer - https://www.jenkins.io/blog/authors/dduportal/[Damien Duportal]
+
+Governance board meetings are held every two weeks. Meeting notes and
+recordings of the meetings are available on the
+https://community.jenkins.io/tag/governance[_Jenkins community site_].
+
+== Security Update
+
+Contributed by: https://github.com/Wadeck[Wadeck Follonier]
+
+* One security advisory during November about plugins
+** Usually the vulnerabilities in third party dependencies are not impacting the final product, it is worth mentioning when there is a true
+positive case.
+Pipeline Utility Steps Plugin was impacted by https://commons.apache.org/proper/commons-configuration/security.html#CVE-2022-33980_prior_to_2.8.0.2C_RCE_when_applied_to_untrusted_input[CVE-2022-33980], leading to a remote code execution in Jenkins.
+
+
+== Infrastructure Update
+
+Contributed by: https://github.com/dduportal[Damien Duportal]
+
+* Upgrade of our controllers to the latest LTS 2.375.1
+* New “jenkinsci” NPM official account for Jenkins project
+* JDK17 for Windows provided to developers
+* https://meetings.jenkins-ci.org/[https://meetings.jenkins-ci.org/] website (archive Board meeting minutes) is recovered and back
+* Deprecated plugin “windows-slaves” removed
+* Azure networks are now code-managed to allow incoming IPv6 support and
+improved network performances
```

Comment:
```suggestion
improved network performances.
```

Reply:
Thank you so much for your review @alyssat 🙏